DESCRIPTION

```diff
@@ -48,6 +48,7 @@ Imports:
   dplyr (>= 1.0.7),
   opalr (>= 3.1.1),
   MolgenisArmadillo (>= 1.1.3),
+  MolgenisArmadillo (< 2.0.0),
```

Comment:
@erikzwart @marikaris When doing the major dsUpload release, make sure to replace these two lines with:

```MolgenisArmadillo (>= 2.0.0),```